Sources/Mantis/Protocols/CropViewControllerDelegate.swift

```diff
@@ -19,18 +19,25 @@ public protocol CropViewControllerDelegate: AnyObject {
     func cropViewControllerDidEndResize(_ cropViewController: CropViewController, original: UIImage, cropInfo: CropInfo)
     
     @available(*, deprecated, message: "Use cropViewControllerDidImageTransformed(_ cropViewController: CropViewController, transformation: Transformation) instead")
-    func cropViewControllerDidImageTransformed(_ cropViewController: CropViewController)
```

Comment:
Can you help to remove this `@available(*, deprecated, message: "Use cropViewControllerDidImageTransformed(_ cropViewController: CropViewController, transformation: Transformation) instead")` too?

It is for the deleted `func cropViewControllerDidImageTransformed(_ cropViewController: CropViewController)
Comment

Thank you very much!`

Reply:
change made